app/models/concerns/operations/location_operations.rb

```diff
@@ -0,0 +1,53 @@
+# Centralizing location operations to facilitate upcoming migration
+# away from packages_locations to packages_inventory
+#
+# @example
+#   Operations::move(2, package_a, from: A, to: B)
```

Comment:
I know this is work in progress but technically this should be only `Operations::move(2, package_a, to: B)`

What I mean to say is it should pick up the location in which is right now by itself.

For example say, Package A is in Location X
Now, someone does `Operations::move(2, package_A, from: Y, to: Z)` which should not be possible as you can't move something which is not at that location. 

What do you think? Correct me if I'm wrong or if I misunderstood this. :)

Reply:
@Darshan-Chauhan since this is made in preparation for the new inventory system, it is designed with the assumption that the package is in multiple places and moved in chunks.

So it evaluates to "Move X of package Y from L1 to L2"

If someone tries to move a package from a location in which it's not, the method will raise an error. Which is perfectly fine :)